### PR TITLE
Add testing documentation

### PR DESCRIPTION
**Pull request by jepoorbaugh**
Adds documentation for running tests locally

### CODE REVIEW

**Review by Jxboss911 — APPROVED** [review]
Looks good

**Review by owenmock22 — CHANGES_REQUESTED** [review]
You do not need a virtual environment to run the test.  It is currently working without one

**Review by Jxboss911 — DISMISSED** [review]

**Review by owenmock22 — DISMISSED** [review]
Looks good.  Good idea adding pycache to git ignore

File: .gitignore [hunk 1]
```diff
@@ -61,3 +61,4 @@ node_modules/prettier/plugins/yaml.mjs
 package-lock.json
 git
 node_modules/
+__pycache__
```
Comment (jncraton) [inline .gitignore:64]: This change is not related to adding testing documentation.

Comment (jepoorbaugh) [reply .gitignore]: This change has now been removed.

File: .gitignore [hunk 2]
```diff
@@ -60,4 +60,4 @@ node_modules/prettier/plugins/yaml.js
 node_modules/prettier/plugins/yaml.mjs
 package-lock.json
 git
-node_modules/
```
Comment (jepoorbaugh) [inline .gitignore:63]: This change is still shown but it makes no changes to the overall file.

**Review by owenmock22 — APPROVED** [review]

**Review by dtlemon — APPROVED** [review]

**Review by AndrewG2026 — APPROVED** [review]
looks good however i think there is a small miss spelling in step two of testing
